compiler/luci/import/src/Nodes/CircleUnidirectionalSequenceLSTM.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Import/Nodes/CircleUnidirectionalSequenceLSTM.h"
+
+#include <luci/IR/Nodes/CircleUnidirectionalSequenceLSTM.h>
+
+#include <loco.h>
+
+namespace luci
+{
+
+bool CircleUnidirectionalSequenceLSTMGraphBuilder::validate(const ValidateArgs &args) const
+{
+  if (args.op.inputs.size() != 24)
+    return false;
+
+  return true;
+}
+
+CircleNode *CircleUnidirectionalSequenceLSTMGraphBuilder::build_node(
+    const circle::OperatorT &op, const std::vector<CircleNode *> &inputs, loco::Graph *graph) const
+{
+  auto *node = graph->nodes()->create<CircleUnidirectionalSequenceLSTM>();
+  node->input(inputs.at(0));
+  node->input_to_input_weights(inputs.at(1)); // Optional
+  node->input_to_cell_weights(inputs.at(2));
```

Comment:
Q) if `input_to_input_weights` didn't exist as it's optional then is this `input_to_cell_weights` be `inputs.at(2)` or `inputs.at(1)` ?
Usually we had optional at the end so I'm not sure existing current codes will work for this PR.
Can you make a recipe for a test that some of the optional nodes in the middle is absent and this works?

Reply:
Okay I will test it on my draft!

Reply:
>Q) if `input_to_input_weights` didn't exist as it's optional then is this `input_to_cell_weights` be `inputs.at(2)` or `inputs.at(1)` ?
@seanshpark as I tested on my draft, I found that inputs have `luci::CircleOutputExclude` on empty input(if options are not provided case). So, inputs.at(2) will be in `input_to_cell_weights`.

Reply:
OK thanks!
Please add your test material (recipe files I guiess) too :)